pandas/tests/reshape/concat/test_concat.py

```diff
@@ -638,3 +638,18 @@ def test_concat_multiindex_with_empty_rangeindex():
     result = concat([df1, df2])
     expected = DataFrame([[1, 2], [np.nan, np.nan]], columns=mi)
     tm.assert_frame_equal(result, expected)
+
+
+def test_concat_posargs_deprecation(all_parsers):
+    # https://github.com/pandas-dev/pandas/issues/41485
+    df = pd.DataFrame([[1, 2, 3]], index=["a"])
+    df2 = pd.DataFrame([[4, 5, 6]], index=["b"])
```

Comment:
pandas/tests/reshape/concat/test_concat.py:645:10: PDF019 found both 'pd.DataFrame' and 'DataFrame' in the same file
pandas/tests/reshape/concat/test_concat.py:646:11: PDF019 found both 'pd.DataFrame' and 'DataFrame' in the same file